lib/improver/tests/database/test_SpotDatabase.py

```diff
@@ -527,6 +528,9 @@ def setUp(self):
                                    "index")
         self.dataframe = self.plugin.to_dataframe(cubes)
 
+    @ManageWarnings(
```

Comment:
We do need to check with Laurence we are happy to suppress this one indefinitely.

Reply:
Checked with Laurence. He is happy to suppress this message.